internal/functions/deploy/bundle.go

```diff
@@ -57,12 +57,16 @@ func (b *dockerBundler) Bundle(ctx context.Context, entrypoint string, importMap
 	if viper.GetBool("DEBUG") {
 		cmd = append(cmd, "--verbose")
 	}
+	env := []string{}
+	if custom_registry := os.Getenv("NPM_CONFIG_REGISTRY"); custom_registry != "" {
+		env = append(env, "NPM_CONFIG_REGISTRY="+custom_registry)
```

Comment:
```suggestion
	if customRegistry := os.Getenv("NPM_CONFIG_REGISTRY"); customRegistry != "" {
		env = append(env, "NPM_CONFIG_REGISTRY="+customRegistry)
```

Just a nitpick on casing